lib/kbuildbarn/README.md

```diff
@@ -0,0 +1,4 @@
+This package is designed for integration between buildbarn and other first party applications written in enkit.
+
+Buildbarn and other remote executors format their external tools around already defined .proto definitions. This library
+is designed to be configurable based on external templates.
```

Comment:
It would be helpful to give a short example of intended API usage in this readme to end up with a []byte slice from the input bytestream:// URL. Application developers are more likely to look here than in the unit test file for this info.

Reply:
Added more docs in both the readme and the New factor constructor